tests/test_wallet.py

```diff
@@ -366,11 +367,16 @@ def test_txprepare(node_factory, bitcoind, chainparams):
     l1.rpc.txdiscard(prep4['txid'])
 
     # Try passing in a utxo set
-    utxos = [utxo["txid"] + ":" + str(utxo["output"]) for utxo in l1.rpc.listfunds()["outputs"]][:4]
-
+    utxos = [utxo["txid"] + ":" + str(utxo["output"])
+             for utxo in l1.rpc.listfunds()["outputs"]][:4]
```

Comment:
Please don't break list-comprehensions across multiple lines. It usually is a sign that a loop should be used instead.